src/components/panel/panel.spec.js

```diff
@@ -2713,6 +2713,70 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('contentElement support: ', function() {
+    var config;
+
+    beforeEach(function() {
+      config = {
+        contentElement: angular.element('<div>'),
+        position: $mdPanel.newPanelPosition().absolute().center()
+      };
+    });
+
+    it('should wrap the content element in the proper HTML and assign ' +
+      'the wrapper to the panel reference', function() {
+        openPanel(config);
+
+        expect(panelRef.panelEl.parent()).toHaveClass(PANEL_WRAPPER_CLASS.substring(1));
```

Comment:
I'd suggest a new line before the `.toHaveClass` in this case to help with readability.